buildpipeline/pipeline.json

```diff
@@ -283,6 +311,75 @@
           }
         },
         {
+          "Name": "DotNet-CoreFx-Trusted-Windows",
+          "Parameters": {
+            "PB_Platform": "arm",
+            "PB_BuildArguments": "-buildArch=arm -Release -portable -- /p:SignType=real /p:RuntimeOS=win10",
+            "PB_BuildTestsArguments": "-buildArch=arm -Release -SkipTests -Outerloop -- /p:RuntimeOS=win10 /p:ArchiveTests=true",
```

Comment:
Not 100% certain but you may need to pass `-portable` to the buildtests as well.